Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [workspace]
 resolver = "2"
 members = [ 
-    "bench-tx",
     "miden-lib",
     "miden-tx",
     "objects",
+    "bin/bench-tx",
+    "bin/miden-tx-prover",
```

Comment:
nit: I'd rename `miden-tx-prover` to just `tx-prover`.

We should also probably rename `bench-tx` to `tx-bench` - but that's for another PR.